### PR TITLE
[#165364032]-fixed the seeding problem with the context

### PR DESCRIPTION
**Pull request by ljmars2351**

### CODE REVIEW

File: src/Housing.Forecast.Context/ForecastContext.cs [hunk 1]
```diff
@@ -34,32 +34,7 @@ public ForecastContext(DbContextOptions<ForecastContext> options) : base(options
...
-            var AddedEntities = ChangeTracker.Entries().Where(E => E.State == EntityState.Added).ToList();
-
-            AddedEntities.ForEach(E =>
-            {
-                E.Property("Created").CurrentValue = DateTime.Now;
-            });
-
-            return base.SaveChangesAsync(cancellationToken);
-        }
+        // Deleted override of the save and saveAsync that just created an
```
Comment (c-troha) [inline src/Housing.Forecast.Context/ForecastContext.cs]: These notes should go in the commit message not the code itself

File: src/Housing.Forecast.Service/Controllers/ForecastController.cs [hunk 2]
```diff
@@ -415,7 +415,8 @@ private async Task<List<Snapshot>> CreateSnapshots(DateTime? date, string locati
         /// <returns>
         /// Returns a new snapshot instantance that was created with the provided data.
         /// </returns>
-        private Snapshot CreateSnapshot(int rooms, int users, string location, int maleCount, int femaleCount, DateTime? date = null)        {
```
Comment (c-troha) [inline src/Housing.Forecast.Service/Controllers/ForecastController.cs]: why did you make this change?

Comment (ljmars2351) [reply src/Housing.Forecast.Service/Controllers/ForecastController.cs]: making things null by default means we have all seeding on today's date.

Comment (c-troha) [reply src/Housing.Forecast.Service/Controllers/ForecastController.cs]: hmm, this isn't gonna break anything then?

**Pull request merged**